src/soc/intel/alderlake/Makefile.mk

```diff
@@ -27,6 +28,7 @@ romstage-y += reset.c
 ramstage-y += acpi.c
 ramstage-y += chip.c
 ramstage-y += cpu.c
+ramstage-y += cpu_util.c
```

Comment:
I think `romstage` is enough.

Reply:
For now yes.

https://github.com/Dasharo/coreboot/pull/518/commits/62fadd264bb206231bd51921bf4cc1a7e1731a3d#diff-3cbb42793ae5edb247eb7ed23b831be01a29436e0c036a68eb83460c6991941aR21